tests/test_cmd_init.py

```diff
@@ -166,14 +175,31 @@ def test_existing_db_error(self):
         self.assertEqual(code1, CMD_SUCCESS)
 
         code2 = self.cmd.initialize(self.name)
-        self.assertEqual(code2, CODE_DATABASE_ERROR)
+        self.assertEqual(code2, CODE_DATABASE_EXISTS)
 
         # Context added to catch deprecation warnings raised on Python 3
         with warnings.catch_warnings(record=True):
             output = sys.stderr.getvalue().strip()
             self.assertRegexpMatches(output,
                                      DB_EXISTS_ERROR % {'database' : self.name})
 
+        cmd = Init(user=self.kwargs['user'], password=self.kwargs['password'],
```

Comment:
Why do you need to create a new `Init` instance? I think assertions from above cover this case. In my opinion, this was already checked.